docker-stack-prod.yml

```diff
@@ -45,7 +45,7 @@ services:
             - back_network
         volumes:
             - /var/run/docker.sock:/var/run/docker.sock
-            - /volumes/whitebox_program_uploads:/uploads:ro
```

Comment:
The `launcher` need to create a subfolder for each new submission in the compilation directory, hence it is supposed to be writable.